docs/roadmap/README.md

```diff
@@ -7,9 +7,7 @@ parent:
 
 # Roadmap
 
-## Roadmap 2.0
-
-For current roadmap, check out the Cosmos 2.0 [Roadmap](./cosmos-hub-roadmap-2.0.md).
+For current roadmap and planned features, check out the Cosmos Hub [Release Log](./cosmos-hub-release-log.md).
 
 ## Cosmos Hub History
```

Comment:
I'd move this table to the cosmos-hub-release-log.md file. This readme.md can be the 'planned features doc'.